docs/tutorialkit.dev/src/content/docs/reference/react-components.mdx

```diff
@@ -116,6 +116,8 @@ A component to list files in a tree view.
   }
   ```
 
+* `allowEditPatterns?: string[]` - Glob patterns for paths that allow editing files and folders. Defaults to `['**']`.
```

Comment:
Default `**` is correct. Editing is disabled by default on `@tutorialkit/astro` level. On `@tutorialkit/react` level, it is enabled when you pass `onFileChange` prop. 

Maybe this could be

```
* `allowEditPatterns?: string[]` - Glob patterns for paths that are allowed to call `onFileChange`. Defaults to `['**']`.
```